.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: ci
+
+on:
+  push:
+    branches: [ master, qa ]
+  pull_request:
+    branches: [ master, qa ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ '1.11.x']
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install
+      env:
+        GO111MODULE: on
```

Comment:
IIUC we need to support gopath users going forward. I assume this means that turning modules OFF. I would check in with DevRel on the status of all this. I believe this was the purpose of the sh files below.

Reply:
I checked with devrel and supporting gomod is the way to go, though gopath is still used by some users due to tech debt. I added tests for both.